apps/web/src/pages/Project/index.tsx

```diff
@@ -47,10 +47,16 @@ function Project() {
         <Header>
           <InfoBox>
             <StyledAvatar
+              onClick={() => {
```

Comment:
could you wrap the icon inside an unstyled button, for accessibility purposes.

Reply:
Ok! I will also add the functionality to navigate back to previous page w/o refreshing